dexbot/cli.py

```diff
@@ -93,7 +93,8 @@ def run(ctx):
             signal.signal(signal.SIGHUP, kill_workers)
             # TODO: reload config on SIGUSR1
             # signal.signal(signal.SIGUSR1, lambda x, y: worker.do_next_tick(worker.reread_config))
-        except ValueError:
+        except AttributeError:
```

Comment:
@thehapax why exception name has been changed? In the future, please do a description in commit messages why you are changing something.

Reply:
See the conversation from Juhani on telegram on March 10 t.me/DEXBOTbts/28573